src/unity/python/turicreate/test/test_object_detector.py

```diff
@@ -220,6 +220,18 @@ def test_predict(self):
         pred0 = self.model.predict(sf[:0])
         self.assertEqual(len(pred0), 0)
 
+    def test_single_image(self):
```

Comment:
Could you also update `test_single_image` in `test_image_classifier.py`?

Reply:
Good idea. I added some checks of the return type.

I also pushed some updates to the docstrings to reflect these changes.